frontend/src/styles/_layout.scss

```
@@ -540,11 +540,32 @@
   font-size: 12px;
-  padding: 0 12px 6px;
+  text-align: left;
+
+  &:hover {
+    background: var(--hover);
+    color: var(--text);
+  }
+
+  .account-toggle-label {
+    @include truncate;
+    flex: 1;
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)